src/features/planning/pages/create-fare-experimental/create-fare-experimental.form.ts

```diff
@@ -0,0 +1,27 @@
+import { FormControl, FormGroup, Validators } from '@angular/forms';
+
+export type CreateFareFormValues = {
```

Comment:
```suggestion
export type CreateFare = {
```
Move in `planning/providers/actions/create-fare-experimental.action.provider.ts` file